app/views/devise/sessions/new.html.haml

```diff
@@ -19,7 +19,7 @@
         = f.label :remember_me
     .col
       = link_to "Login problems?", new_password_path(resource_name), class: "pull-right"
-      - if devise_mapping.omniauthable?
```

Comment:
This is not necessary nor desirable. Commenting the omniauth config in the device page will do the trick.